packages/client/src/entities/Wallet.ts

```diff
@@ -0,0 +1,42 @@
+import { transferMoney } from "..";
+import { Configuration, HTTPHeaders, WalletsApi } from "../api-client";
+import { generateAddressFrom } from "../functions";
+
+import { Address } from "./Address";
+import { Amount } from "./Amount";
+import { Description } from "./Description";
+import { PrivateKey } from "./PrivateKey";
+
+
+export class Wallet {
+    private privateKey: PrivateKey;
+    private wallet: WalletsApi;
+
+    constructor(config: { host: string; port: number; privateKey: PrivateKey; headers: HTTPHeaders }) {
+        this.privateKey = config.privateKey;
+
+        const configuration = new Configuration({
+            basePath: `${config.host}:${config.port}`,
+            headers: config.headers,
+        });
+
+        this.wallet = new WalletsApi(configuration);
+
+    }
+
+    async sendMoney(to: Address, amount: Amount, description: Description) {
```

Comment:
Tokens dont always represent money
```suggestion
    async sendTokens(to: Address, amount: Amount, description: Description) {
```